frontend/packages/console-shared/src/components/formik-fields/SyncedEditorField.tsx

```diff
@@ -29,6 +35,7 @@ type SyncedEditorFieldProps = {
   name: string;
   formContext: EditorContext<SanitizeToForm>;
   yamlContext: EditorContext<SanitizeToYAML>;
+  lastViewedEditorTypeUserSettingKey: string;
```

Comment:
Can we keep the prop name shorter? Maybe just `userSettingsKey`?

Reply:
could we use `lastViewUserSettingKey`?
WDYT?